workflow-trace-viewer/src/jvmMain/kotlin/com/squareup/workflow1/traceviewer/model/WorkflowNode.kt

```diff
@@ -0,0 +1,14 @@
+package com.squareup.workflow1.traceviewer.model
+
+/**
+ * Since the logic of Workflow is hierarchical (where each workflow may have parent workflows and/or
+ * children workflows, a tree structure is most appropriate for representing the data rather than
+ * using flat data structures like an array.
+ *
+ * TBD what more metadata should be involved with each node, e.g. (props, states, # of render passes)
+ */
+public data class WorkflowNode(
```

Comment:
1. Let's avoid reusing the term `WorkflowNode`, that's a very key object in the workflow runtime. Maybe just `Node`? Or `ViewerNode`?

2. Avoid `data class` in library APIs

3. A package with a single member is odd, unless you're very confident that there will be other classes here soon. Perhaps you should promote this class and just call it `Model` or `NodeModel`.

Be interested to hear @steve-the-edwards's take.